modules/examples/src/main/java/com/ibm/cloud/platform_services/resource_manager/v2/ResourceManagerExamples.java

```diff
@@ -0,0 +1,186 @@
+        .id(resourceGroupId)
+        .build();
+
+      Response<ResourceGroup> response = resourceManagerService.getResourceGroup(getResourceGroupOptions).execute();
+      ResourceGroup resourceGroup = response.getResult();
+
+      System.out.println(resourceGroup);
+      // end-get_resource_group
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+          e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-update_resource_group
+      UpdateResourceGroupOptions updateResourceGroupOptions = new UpdateResourceGroupOptions.Builder()
+        .id(resourceGroupId)
+        .name("RenamedExampleGroup")
+        .state("ACTIVE")
+        .build();
+
+      Response<ResourceGroup> response = resourceManagerService.updateResourceGroup(updateResourceGroupOptions).execute();
+      ResourceGroup resourceGroup = response.getResult();
+
+      System.out.println(resourceGroup);
+      // end-update_resource_group
+    } catch (ServiceResponseException e) {
+        logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+          e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-list_resource_groups
+      ListResourceGroupsOptions listResourceGroupsOptions = new ListResourceGroupsOptions.Builder()
+              .accountId(userAccountId)
+              .includeDeleted(true)
+              .build();
+
+      Response<ResourceGroupList> response = resourceManagerService.listResourceGroups(listResourceGroupsOptions).execute();
+      ResourceGroupList resourceGroupList = response.getResult();
+
+      System.out.println(resourceGroupList);
+      // end-list_resource_groups
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                                 e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-delete_resource_group
+      DeleteResourceGroupOptions deleteResourceGroupOptions = new DeleteResourceGroupOptions.Builder()
+              .id(resourceGroupId)
+              .build();
+
+      deleteResourceManagerService.deleteResourceGroup(deleteResourceGroupOptions).execute();
+      // end-delete_resource_group
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s\nError details: %s",
+                                 e.getStatusCode(), e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-list_quota_definitions
+      ListQuotaDefinitionsOptions listQuotaDefinitionsOptions = new ListQuotaDefinitionsOptions();
+
+      Response<QuotaDefinitionList> response = resourceManagerService.listQuotaDefinitions(listQuotaDefinitionsOptions).execute();
```

Comment:
Because there are no parameters for this operation (and hence no fields in the options model class), you could invoke the no-arg verson of the listQuotaDefinitions() function (and then remove the construction of the options model instance above.
```suggestion
      Response<QuotaDefinitionList> response = resourceManagerService.listQuotaDefinitions().execute();
```